api/HTMLVideoElement.json

```diff
@@ -610,6 +610,57 @@
             "deprecated": true
           }
         }
+      },
+       "msIsStereo3D": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/msIsStereo3D",
+          "support": {
```

Comment:
Sort the `support` block alphabetically

Reply:
Not quite sure what you mean... seems to match the other property entries...?

Reply:
Generally it's supposed to be alphabetical, though it hasn't really been enforced in the past. This file has weird ordering, not sure what's up with it.

Reply:
@mattwojo See issue #398 and pull request #1882.